src/components/SearchCode.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <b-tabs>
+    <b-tab title="Python">
+      <CodeBox :code="pythonCode" language="python" />
+    </b-tab>
+    <b-tab title="Javascript">
+      <CodeBox :code="javascriptCode" language="javascript" />
+    </b-tab>
+  </b-tabs>
+</template>
+    
+  <script>
+    import { BTabs, BTab } from 'bootstrap-vue';
+    export default {
+      name: "SearchCode",
+      components: {
+        BTab,
+        BTabs,
+        CodeBox: () => import('./CodeBox.vue'),
+      },
+      props: {
+        catalogHref: {
+          type: String,
+          default: '',
+        },
+        filters: {
+          type: Object,
+          default() {
+             return {};
+           },
+        },
+      },
+      data() {
+        return {
+          componentId: `${this.language}Content`,
+          pythonCode: null,
+          javascriptCode: null,
+          rCode: null
+        };
+      },
+      watch: {
+        filters: {
+          deep: true,
+          handler() {
+            this.updateCode();
+          }
+        }
+      },
+      created() {
+        this.updateCode();
+      },
+      methods: {
+        dedent(str) {
```

Comment:
This dedents a bit too much, the JS code is not indented at all, which looks rather weird.